docs/sql/statements/start-transaction.rst

```diff
@@ -0,0 +1,42 @@
+.. highlight:: psql
+.. _ref-start:
+
+=====================
+``START TRANSACTION``
+=====================
+
+Start a transaction block
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+::
+
+   START TRANSACTION [ transaction_mode [ , ...] ]
+
+where ``transaction_mode`` is one of::
```

Comment:
```suggestion
Where ``transaction_mode`` is one of::
```

I think the SQL synopsis sections, in general, could do with a bit of reworking to expand the explanation of what is being documented. however, this probably _isn't_ the PR to start rethinking that

added this suggestion in the meantime as a minor, easy thing to improve